src/main/webapp/resources/js/pages/admin/components/announcements/CreateAnnouncementModal.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useRef } from "react";
+import { Checkbox, Form, Input, Modal } from "antd";
+import { IconEdit } from "../../../../components/icons/Icons";
+import { FONT_COLOR_PRIMARY } from "../../../../styles/fonts";
+import { MarkdownEditor } from "../../../../components/markdown/MarkdownEditor";
+
+/**
+ * React component to display an Ant Design Modal containing a form to create a new announcement
+ * @param {boolean} visible - whether the modal is open or not
+ * @param {function} closeModal - the function to close the modal
+ * @param {function} createAnnouncement - the function to create a new announcement and refresh the table
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function CreateAnnouncementModal({
+  visible,
+  closeModal,
+  createAnnouncement,
+}) {
+  const markdownRef = useRef();
+  const [form] = Form.useForm();
+
+  const onCancel = () => {
+    form.resetFields();
+    closeModal();
+  };
+
+  function saveMarkdown() {
+    form.validateFields().then((values) => {
+      const markdown = markdownRef.current.getMarkdown();
+      const title = values.title;
+      const priority = values.priority;
+
+      createAnnouncement(title, markdown, priority);
+      closeModal();
+    });
+  }
+  return (
+    <Modal
+      visible={visible}
+      title={
+        <>
```

Comment:
Instead of a react fragment, let's use `Space` to give some breathing room between the icon and the title.
```javascript
title={
        <Space>
          <IconEdit style={{ color: FONT_COLOR_PRIMARY }} />
          {i18n("CreateNewAnnouncement.title")}
        </Space>
      }
```

Reply:
Please see 3bdf855381a0c447dffa1528a00bad9702726e58